project-base/app/tests/FrontendApiBundle/Test/CommonGraphQlWithLoginTestCase.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Test;
+
+abstract class CommonGraphQlWithLoginTestCase extends GraphQlTestCase
+{
+    public const string DEFAULT_USER_EMAIL = 'no-reply@shopsys.com';
+    public const string DEFAULT_USER_PASSWORD = 'user123';
+
+    private string $currentAccessToken = '';
+
+    /**
+     * @param string|null $customerUserEmail
+     * @param string|null $customerUserPassword
+     * @return string
+     */
+    private static function getLoginQuery(
+        ?string $customerUserEmail = null,
+        ?string $customerUserPassword = null,
+    ): string {
+        $customerUserEmail = $customerUserEmail ?? self::DEFAULT_USER_EMAIL;
+        $customerUserPassword = $customerUserPassword ?? self::DEFAULT_USER_PASSWORD;
+
+        return '
```

Comment:
nitpick. we may use `project-base/app/tests/FrontendApiBundle/Functional/Login/graphql/LoginMutation.graphql`

Reply:
good point, fixed in 120747299f